src/plugins/utils/performance-utils.ts

```diff
@@ -32,3 +32,14 @@ export const defaultPerformancePluginConfig = {
     ],
     sampleTypes: [ResourceType.IMAGE, ResourceType.OTHER]
 };
+
+/** Field renderBlockingStatus is currently missing from the node runtime and will cause build failures. */
+export interface PerformanceResourceTimingPolyfill
+    extends PerformanceResourceTiming {
+    renderBlockingStatus?: string;
+}
+
+export interface OmittedResourceFields {
```

Comment:
question: What happens if we don't propagate these fields? My intuition is that we can do without them.

Reply:
Same as above, removed.